MeetupPlatformApi/Persistence/Migrations/20220408150748_AddUsersByRoles.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace MeetupPlatformApi.Migrations
+{
+    public partial class AddUsersByRoles : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<string>(
+                name: "role",
+                table: "users",
+                type: "text",
+                nullable: false,
+                defaultValue: "");
+
+            migrationBuilder.AddColumn<Guid>(
+                name: "organizer_id",
+                table: "meetups",
+                type: "uuid",
+                nullable: false,
+                defaultValueSql: "gen_random_uuid()");
```

Comment:
Wrong, meetup should not exist without an organizer and should not contain some random id instead of the real organizer's id. Consider deleting all meetups from a db